### ISSUE TITLE
V4

### ISSUE THREAD

**Issue by josh-onchain** [PondFive/TerraformFargateSeleniumGrid#1]
Have you tried this with Version4 of Selenium?

**Comment by pgsung** [comment]
Yes I tried it with following changes and it works. https://github.com/PondFive/TerraformFargateSeleniumGrid/pull/3
